src/VisualStudio/Core/Def/ServicesVisualStudio.csproj

```diff
@@ -679,6 +693,12 @@
       <SubType>Designer</SubType>
       <LastGenOutput>ServicesVSResources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
+    <EmbeddedResource Include="VSPackage.resx">
```

Comment:
This is the resource that merges in the .vsct file. Personal superstition insists the resource name must be VSPackage, but I could be wrong. Maybe @pharring knows?